packages/api/src/modules/notify/@types/NotificationDataType.ts

```diff
@@ -0,0 +1,25 @@
+import { NotificationType } from "./NotificationType";
+
+export interface NotificationDataType {
```

Comment:
```suggestion
export interface NotificationDataTypes {
```

Je trouve le nom au pluriel + adapté si je comprends bien ce que ça fait